### DISCUSSION TITLE
Release next version of package to Nuget

### DISCUSSION THREAD

**Discussion by mishamyte** [jbogard/MongoDB.Driver.Core.Extensions.DiagnosticSources#20] [Q&A] [answered]
Hi @jbogard,

Could you please publish the vNext version of package to Nuget, so it could be used publicly.

**Comment by jbogard** [comment, marked as answer]
I'll have some time next week.

**Comment by mishamyte** [comment]
Hi @jbogard,

Any updates for this?

**Comment by jbogard** [comment]
Sorry, I was off by a week, I'll have time next week

**Reply by mishamyte** [reply]
Hey there!

I'm sorry, don't want to be annoying, but any chance to find the new version in release soon?
Just wanna use the package

**Reply by mishamyte** [reply]
Thank you for the new version!
